Source/panels/spell_list.cpp

```diff
@@ -200,22 +200,22 @@ std::vector<SpellListItem> GetSpellListItems()
 	int x = mainPanelPosition.x + 12 + SPLICONLENGTH * SPLROWICONLS;
 	int y = mainPanelPosition.y - 17;
 
-	for (int i = RSPLTYPE_SKILL; i < RSPLTYPE_INVALID; i++) {
+	for (int i = static_cast<uint8_t>(SpellType::Skill); i < static_cast<uint8_t>(SpellType::Invalid); i++) {

```

Comment:
we should maybe add a first/last alias

Reply:
> we should maybe add a first/last alias

That'd be good, enum_traits.h defines a couple of template functions that could be used to iterate over enum values if the aliases use `FIRST`/`LAST` as the identifiers.